app/src/main/java/io/pslab/activity/MainActivity.java

```diff
@@ -281,16 +281,16 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                         startActivity(new Intent(MainActivity.this, DataLoggerActivity.class));
                         break;
                     case R.id.nav_share_app:
-                        try {
```

Comment:
@neel1998 I have checked the code and I don't think it's throwing an exception, and as well as android studio is not suggesting any try-catch block so I don't it is required!

Reply:
@abhinavraj23 ACTION_SEND intent throws exception when there are no sharing apps available on device. I know it is a rare case. But I don't see any loss in keeping the try catch block. It will make the app fail-safe. Still this is just a suggestion, I don't mind if mentors decide other way.

Reply:
@neel1998 I think there will be any device where they don't have the default messaging app, they will atleast have an option of copy to clipboard, thus it will work without try-catch @CloudyPadmal @harsh-2711, please suggest

Reply:
@abhinavraj23  okay